Gordon360/Models/ViewModels/RecIM/ParticipantViewModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Gordon360.Models.CCT;
+using Microsoft.Graph;
+using System;
+using System.Collections.Generic;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class ParticipantViewModel
+    {
+        public string ADUserName { get; set; }
```

Comment:
```suggestion
        public string Username { get; set; }
```
My personal preference is username for this type of field. At the very least, we shouldn't add yet another name for this field in the API. We already have ad_username and username used elsewhere.